### ISSUE TITLE
Passing additional options to sauce connect

### ISSUE THREAD

**Issue by ryudice** [zalando/zalenium#332]
Is there a way to pass additional command line arguments to Sauce Connect? I'm having issues with resolving internal DNS domains.

**Comment by diemol** [comment]
Hi @ryudice,

Not really, but in theory any needed parameter could be passed to the container via an env var and then mapped to the binary when starting it. This [file](https://github.com/zalando/zalenium/blob/master/scripts/start-saucelabs.sh) is the one starting it.

What parameter do you have in mind?

**Comment by ryudice** [comment]
Hi @diemol, the DNS parameter:
![image](https://user-images.githubusercontent.com/125211/33030490-09187bde-cde1-11e7-9934-de9d207a83b0.png)

**Comment by diemol** [comment]
OK, this should not be that complicated to add.
Do you feel like adding it via a PR? I could tell you where to get started and how to test it.

Otherwise I can take a look at it in 4-6 weeks since we are working in different stuff now.

**Comment by diemol** [comment]
Hi @ryudice,

This was just implemented in #393, we'll release it tomorrow so you can use it.

You'd use it like this (note the **SAUCE_TUNNEL_DNS** var):

```
docker run --rm -ti --name zalenium -p 4444:4444 \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v /tmp/videos:/home/seluser/videos -e SAUCE_USERNAME -e SAUCE_ACCESS_KEY \
    -e SAUCE_TUNNEL_DNS="8.8.8.8,8.8.4.4:53" \
    --privileged dosel/zalenium start --sauceLabsEnabled true --startTunnel true
```

**Comment by diemol** [comment]
Hi @ryudice,

This was just released, please try it with the latest tag https://github.com/zalando/zalenium/releases/tag/3.8.1f

Feel free to reopen in case something related is not working. 

PS: Don't forget to star the repo to support us!
